bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -133,7 +137,7 @@ public Response getInterpreter(
     }
 
     @POST
-    @Path("/interpreters/{id: [a-zA-Z_0-9]+}")
+    @Path("/interpreters/{ids: [a-zA-Z_0-9,]+}")
     @Consumes(MediaType.TEXT_PLAIN)
     @Operation(operationId = "interpretText", summary = "Sends a text to a given human language interpreter.", responses = {
```

Comment:
You could adjust the summary with "Sends a text to given human language interpreter(s)."